util/provision-run/files/google-cloud-platform-credentials.json.encrypted

```diff
@@ -0,0 +1,121 @@
+63616338666233666635323862373336363865333063613961633161386165663564313362346432
+30363231343930626161373065323162363836666136373063363965313763376130623765653762
+33383833336539376161633832323633616566353561333131623938373436643961353265656139
+37303737633439613262636664346639626536396236373263323132333838346164643435646236
+39323539383631636630646534626232373336336631316636613831663630343630626164373933
+32366662666265363531376162653863663033373037323337333331353930396564653932333135
+61386564323062616565616464323134303338653736346439366135383061373736313233326631
+65366135613962656661656466313264353161343163353365333439353638383039623261396437
+62343233326162376465303866353833656431366161303932653266646138373766623835353164
+64663635623136666338613161366536343833636664363464363965666638616136386333323038
+38346566633265366565346365343539623038626162383063376437323261386165643938346138
+39386232316139376133613066613530356361353765356431656533383563353536323334623962
+62346563633365643231356430323337393135336162366538613130326631373632343730626536
+63336239633334666130623639396363323732306339356330376566353663353064316137383766
+35663362393132636266396337363630616364346632333262306132653934663261386462336563
+65336233636236313637336638643261653161346663393335346231373537373932393632613532
+35653637306430366632373930376336326466323538636536373639333764363966353938383332
+63326435356233306638323161643064303037666362363634363338636561346266353739633138
+34353062326164383135663539393133666232323530323738313237626163373037373839646361
+39343730646361306434303464316362393933393363313837656363396534383031336663366632
+61613632353235343366643130373535336430623734656266393034326664643231666163333165
+31623063663832396438633061623065383933353334636364666464373262396632326532363865
+38653835393965373731663062343033376238393436343861656330396530643637623033323037
+39333433336436323835653466386136613465346566393262633733313739336433373036616362
+63636439326239366161366233346132356361343663623730633535333739363733396664663832
+65383339383132653538633265386361333932386336383431666662353131323738666232306364
+35343264383638626661343162663539616233636435313964343631326562356164633332326333
+66623733656336386534623339333236643061636630646238646462633738376536356264363830
+33616561323433663236383334353164626266626632316439643531336565396634653235386236
+34633134656431613231353037613137633063343534333533636265356438346532373532633439
+30386436626164656230373433646665633034343837303436313034373231616536333964343631
+33636664366431633262383333663934313430616530353734393034363965623636663635366435
+30633433336530643966373931376563323033643265656334353438303037353365623038643362
+64393737396238633937356666613939396635636634653939393232383737323039313239383162
+37373162316434616166343432656135363366313630616262386631623437333933313531613938
+62613233303235666664363635613264383162313538626263303363303131373732396162333061
+30613238353132393162
```

Comment:
Was this intentional?

Reply:
Are you asking about the file generally? It works on my (virtual) machine.

Reply:
Sorry, I should've been more specific. What I meant to ask was: was committing this file intentional? But I feel as though you've indirectly answered that question now, so I will assume the answer is "yes"

Reply:
No need to assume: I'll elaborate. I've committed this file so that we can more effectively share the responsibility of deployment. Deploying this project to production relies on a bunch of sensitive information. Rather than maintaining all that data as separate files, using Ansible Vault in this way allows us to share a single value (the Ansible Vault password) and then dynamically expand all the sensitive information we might need (since it's right here in the repository).

I wrote a little bit about this into the `README.md` file I have submitted, but if you have thoughts on how I could make that less confusing, I'm happy to improve that.

Reply:
I think I just hadn't yet reached that file in the PR file list